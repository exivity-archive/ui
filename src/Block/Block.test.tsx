import React from 'react'
import renderer from 'react-test-renderer'
import { Block } from '.'

// Modify or remove, stories are snapshotted automatically
test('renders Block without props', () => {
  const component = renderer.create(<Block />)
  expect(component.toJSON()).toMatchSnapshot()
})
