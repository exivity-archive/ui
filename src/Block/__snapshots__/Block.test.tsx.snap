// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Block without props 1`] = `
<div
  className=""
/>
`;
